install/installer/pkg/components/server/configmap.go

```diff
@@ -287,7 +287,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		InactivityPeriodForReposInDays: inactivityPeriodForReposInDays,
 		PATSigningKeyFile:              personalAccessTokenSigningKeyPath,
 		Admin: AdminConfig{
-			GrantFirstUserAdminRole: true, // existing default
+			GrantFirstUserAdminRole: false, // Use AdminCredentialsPath to seed the installation with admin credentials
```

Comment:
I wonder if we could not completely remove the "first-user-admin" logic? It's only used in one place

Reply:
I guess we still use this in our preview envs.

Reply:
This change would also disable it in preview env, it's not exposed as installer config currently. There's a follow-up PR which removes it entirely https://github.com/gitpod-io/gitpod/pull/17301, but it was split it because I thought we'd want to wait with the removal until later

Reply:
Ok, I guess we can manually add the `admin` role in case we need it in preview envs